### ISSUE TITLE
Reproduce problem

### ISSUE THREAD

**Issue by cyinen** [ArnabKumarRoy02/ResEmoteNet#7]
on RAF-DB
<img width="537" alt="image" src="https://github.com/user-attachments/assets/0e8682b5-17c3-4387-b2aa-d77e0f27a400">

**Comment by zerooooone** [comment]
I trained the model from scratch and got similar results, with low accuracy. Do you have a good accuracy rate now?

![image](https://github.com/user-attachments/assets/b33854a8-893e-4173-a68e-fe2a9153a08c)

**Comment by windJRK** [comment]
How is it? Do you get better result?

**Comment by SpyrosGew** [comment]
Run the model on FER2013 from kaggle the results are disappointing at least 
![screen](https://github.com/user-attachments/assets/15ccaf3e-a85c-436b-b5c6-1bcebe971ea9)

**Comment by ArnabKumarRoy02** [comment]
For anyone who is trying to reproduce the results, have you tried data augmentation?

If you check all the datasets that were used, you will find them to have class imbalance. It is recommended to augment them in order to balance the classes. Please use different augmentation techniques to increase the samples with lower count than others and try to balance out the dataset.

**Comment by cyinen** [comment]
> I think the authors put the accuracy of the training set in the paper, and the model looks so simple that I initially wondered why such a simple model could achieve such a high accuracy rate

agree with you
